From f8bbcefe3eed253cda0c86a4c2443292beca82ee Mon Sep 17 00:00:00 2001
From: Gary Lin <glin@suse.com>
Date: Thu, 28 Mar 2019 15:26:28 +0800
Subject: [PATCH] Change the path to shim.efi

For openSUSE/SLE, we use shim.efi instead of shimx64.efi.

Signed-off-by: Gary Lin <glin@suse.com>
---
 plugins/uefi/fu-uefi-bootmgr.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Index: fwupd-1.5.2/plugins/uefi/fu-uefi-bootmgr.c
===================================================================
--- fwupd-1.5.2.orig/plugins/uefi-capsule/fu-uefi-bootmgr.c
+++ fwupd-1.5.2/plugins/uefi-capsule/fu-uefi-bootmgr.c
@@ -274,7 +274,8 @@ fu_uefi_bootmgr_bootnext (FuDevice *devi
 	secure_boot = fu_efivar_secure_boot_enabled ();
 	if (secure_boot) {
 		/* test to make sure shim is there if we need it */
-		shim_app = fu_uefi_get_esp_app_path (device, esp_path, "shim", error);
+		shim_app = g_strdup_printf ("%s/shim.efi",
+					    fu_uefi_get_esp_path_for_os (device, esp_path));
 		if (shim_app == NULL)
 			return FALSE;
 
